uber/templates/registration/staffers.html

```diff
@@ -4,36 +4,21 @@
 <ol class="breadcrumb">
   <li><a href="../accounts/homepage">Home</a></li>
   <li>People</li>
-  <li class="active">Staffers{% if search_results %} (Search Results){% endif %}</li>
+  <li class="active">Staffers</li>
 </ol>
 
-{% if search_text %}
-    <div class="panel panel-info">
-        <a href="index?order={{ order }}">Click here</a> to view full volunteer list instead of only search results.
-    </div>
-{% endif %}
```

Comment:
I removed the searching here for two main reasons:
1) There's already an in-page ``DataTable`` search on this page.
2) The more powerful searching is still available on the main Attendee page.

In theory the optimizations we did here would still work with the ``session.search`` method, so we can add it back in if we find there's a use case for it.  I'm not sure there is though, and honestly we should consider removing this page from the default main admin menu anyway, since I think only Stops should really be using it at this point.  But that's all part of a separate discussion at this point.